rdflib/plugins/shared/jsonld/util.py

```diff
@@ -16,14 +16,14 @@
 from posixpath import normpath
 
 from urllib.parse import urljoin, urlsplit, urlunsplit
+from rdflib.parser import create_input_source, BytesIOWrapper, PythonInputSource
```

Comment:
duplicate imports on this line

Reply:
Thanks, likely unnoticed in the previous merge, Deduped now.